src/Model/Aggregate.php

```diff
@@ -34,6 +35,10 @@
  *
  * You can also pass seed (for example field type) when aggregating:
  * $aggregate->groupBy(['first','last'], ['salary' => ['sum([])', 'type'=>'money']];
+ *
+ * @property \atk4\data\Persistence\Sql $persistence
+ *
+ * @method Expression expr($expr, array $args = []) forwards to Persistence\Sql::expr using $this as model
```

Comment:
please see a lot of issues with Model::expr, there are not visible as I supressed them temporary to keep CI green

I think we should:
- stop using dynamic methods, at least for the commonly used names
- in Model, implemented them like:
```
function expr(params) {
    if ($this->persistence->supportsExpr()) {
        return $this->persistence->expr($this, params);
    }

    throw new Exception('Persistence does not support this operation');
}
```

I belive this it the only way to make static analyser happy.

Reply:
For sure we can thinkof something but not in this PR